weld/llvm.rs

```diff
@@ -526,8 +533,17 @@ impl LlvmGenerator {
         let bound_cmp = ctx.var_ids.next();
         let mut grain_size = DEFAULT_GRAIN_SIZE;
         if par_for.innermost {
-            ctx.code.add(format!("{} = icmp ule i64 {}, {}", bound_cmp, num_iters_str, grain_size));
-            ctx.code.add(format!("br i1 {}, label %for.ser, label %for.par", bound_cmp));
+            // Determine whether to always call parallel, always call serial, or
+            // choose based on the loop's size.
+            if par_for.parallelize {
+                ctx.code.add(format!("br label %for.par"));
+            }
+            else if self.multithreaded {
```

Comment:
move this up a line so that it's on the same line as the previous closing curly brace

Reply:
to make this code clearer i would actually have
`if par_for.parallelize {
...
} else {
  if (self.multithreaded) {
  ...
  } else {
  ...
  }
}`